imports/ui/clients/clientTagList.js

```diff
@@ -14,12 +15,16 @@ Template.clientTagListView.helpers({
     return ClientTags.find().fetch();
   },
   newClientTagHandler() {
-    return (data) =>
-      Meteor.call('tagApi', 'createClientTag', { operation: 1, ...data }, (err, res) => {
-        if (!err) ClientTags._collection.insert({ _id: res.id, ...res }); // eslint-disable-line
-      });
+    return (res) => ClientTags._collection.insert({ _id: res.id, ...res }); // eslint-disable-line
```

Comment:
I remember I had errors when using `xyz._collection.insert` when used in conjunction with publish function that uses `this.added`. calling this.added for a document that was previously inserted on a client side.